system_setup/common/wsl_utils.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# Copyright 2015-2021 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import subprocess
+
+log = logging.getLogger("subiquity.system_setup.common.wsl_utils")
+
+config_ref = {
+    "wsl": {
+        "automount": {
+            "enabled": "automount",
+            "mountfstab": "mountfstab",
+            "root": "custom_path",
+            "options": "custom_mount_opt",
+        },
+        "network": {
+            "generatehosts": "gen_host",
+            "generateresolvconf": "gen_resolvconf",
+        },
+        "interop": {
+            "enabled": "interop_enabled",
+            "appendwindowspath": "interop_appendwindowspath",
+        }
+    },
+    "ubuntu": {
+        "GUI": {
+            "theme": "gui_theme",
+            "followwintheme": "gui_followwintheme",
+        },
+        "Interop": {
+            "guiintegration": "legacy_gui",
+            "audiointegration": "legacy_audio",
+            "advancedipdetection": "adv_ip_detect",
+        },
+        "Motd": {
+            "wslnewsenabled": "wsl_motd_news",
+        }
+    }
+}
+
+
+def get_windows_locale():
+    try:
+        process = subprocess.run(["powershell.exe", "-NonInteractive",
+                                  "-NoProfile", "-Command",
+                                  "(Get-Culture).Name"],
+                                 capture_output=True)
+        if process.returncode != 0:
+            log.error("cannot determine Windows locale, fallback to derault: ")
+            return None
+        else:
```

Comment:
Style:
```
if <xxx>:
    return None
```
-> align back `tmp_code = process.stdout.rstrip().decode("utf-8")` with the if: as you returned in the if, the "else" statement is not needed